.travis.yml

```diff
@@ -40,8 +40,10 @@ before_script:
 
 script:
   - "./datastore/manage.py migrate"
-  - "./datastore/manage.py test tests"
+  - "coverage run --source='./datastore' ./datastore/manage.py test tests"
   - "datagetter.py --help" # Verify it compiles and is installed
   - "flake8"
-  - "isort --check-only --recursive ./"
```

Comment:
Why remove recursive?

Reply:
@michaelwood was a suggestion of James. He said that it should be deleted. (I can't tag him here).